include/mbgl/gfx/rendering_stats.hpp

```diff
@@ -16,7 +16,22 @@ struct RenderingStats {
     int memTextures;
     int memIndexBuffers;
     int memVertexBuffers;
+
+    RenderingStats& operator+=(const RenderingStats& right);
 };
 
+inline RenderingStats& RenderingStats::operator+=(const RenderingStats& r) {
```

Comment:
Some of these additions might not provide very useful results. For example `numCreatedTextures` also tracks the number of textures in the texture pool.

Reply:
@mpulkki-mapbox shall we just skip it?

Reply:
Discussed offline: it's looks OK to summarize all numbers given that we simulate rendering each tile in isolation (e.g. in a separate process)